### PR TITLE
Added Advanced Computer Player. Changed Game and GameOption to integr…

### PR DESCRIPTION
**Pull request by LeanderDiazBone**
Ready for integration for two different kind of Computer Players with different strengths.

### CODE REVIEW

File: Assets/Scripts/ComputerPlayer.cs [hunk 1]
```diff
@@ -3,71 +3,15 @@
...
-                    for (int n = 0; n < numberPool.Count; n++){
-						if (game.IsValid((i, j), n, id == 1) > 0) return ((i, j), n);
-					}
-				}
-            }
-        }
-        return ((-1,-1),0);
+		List<((int, int), int)> posMoves = PossibleMoves();
+		int randMove = random.Next(0, posMoves.Count);
+		return posMoves[randMove];
```
Comment (yschimpf) [inline Assets/Scripts/ComputerPlayer.cs:14]: Could still think about making even the simple player little more advanced but good for now.

File: Assets/Scripts/ComputerPlayerAdvanced.cs
Comment (yschimpf) [inline Assets/Scripts/ComputerPlayerAdvanced.cs:1]: Follows coding styles. Is the same functionality as before which is already tested.

File: Assets/Scripts/ComputerPlayer.cs (hunk 1 above)
Comment (LeanderDiazBone) [reply Assets/Scripts/ComputerPlayer.cs:14]: Thank you for the nice review.

File: Assets/Scripts/Game.cs
Comment (yschimpf) [inline Assets/Scripts/Game.cs:1]: Addition of advanced computer player as gamemode seems to be fine

File: Assets/Scripts/GameOption.cs
Comment (yschimpf) [inline Assets/Scripts/GameOption.cs:1]: Addition of advanced computer player to gameoption also seems fine

**Review by yschimpf — APPROVED** [review]
Everything seems fine and follows the coding styles. Integration of the advanced computer player is approved.

**Pull request merged**